### PR TITLE
updating certain frontend elements

### PR DESCRIPTION
**Pull request by Pjewels**
1. Added another favicon line for cross browser compatibility 
2. Updated certain descriptions on homepage
3. Changed deploy form placeholders to reflect Starknet

### CODE REVIEW

File: frontend/src/pages/Home/index.tsx [hunk 1]
```diff
@@ -33,12 +33,12 @@ export default function HomePage() {
       <Column as="article" className={styles.secondArticle}>
         <Box>
           <Text.Custom color="text2" marginLeft="8" fontWeight="normal" fontSize="18">
-            Meet Unrrugable Memecoin
+            Meet Unruggable Memecoins
```
Comment (abdelhamidbakhta) [inline frontend/src/pages/Home/index.tsx:36]: the name should be Unruggable Meme now

File: frontend/src/pages/Home/index.tsx [hunk 2]
```diff
@@ -33,12 +33,12 @@ export default function HomePage() {
...
           </Text.Custom>
 
           <Text.Body>
             Tired of getting rugpulled? Introducing Unruggable Memecoin, a project designed with security and
-            transparency at its core.
+            transparency at its core. Your go-to platform for deploying secure memecoins on starknet.
```
Comment (abdelhamidbakhta) [inline frontend/src/pages/Home/index.tsx:41]: change secure to safer

File: frontend/src/pages/Launch/index.tsx [hunk 3]
```diff
@@ -68,23 +68,23 @@ export default function LaunchPage() {
           <Column gap="20">
             <Column gap="4">
               <Text.Body className={styles.inputLabel}>Name</Text.Body>
-              <Input placeholder="Dogecoin" {...register('name')} />
+              <Input placeholder="Starknet" {...register('name')} />
```
Comment (abdelhamidbakhta) [inline frontend/src/pages/Launch/index.tsx:71]: the placeholder should not be Starknet and STRK

**Review by abdelhamidbakhta — CHANGES_REQUESTED** [review]
left some comments

File: frontend/src/pages/Launch/index.tsx (hunk 3 above)
Comment (Pjewels) [reply frontend/src/pages/Launch/index.tsx]: made the changes

**Review by abdelhamidbakhta — APPROVED** [review]
lgmt

**Pull request merged**